doc/INSTALL.md

```diff
@@ -66,7 +66,6 @@ Add the path where the library has been installed
 
 and run `ldconfig` as root.
 
-
```

Comment:
That looks more like a configuration explanation, which doesn't seem to fit in the INSTALL.md?